Source/WebCore/loader/SubframeLoader.cpp

```diff
@@ -328,6 +329,15 @@ RefPtr<LocalFrame> FrameLoader::SubframeLoader::loadSubframe(HTMLFrameOwnerEleme
 
     frame->protectedLoader()->loadURLIntoChildFrame(url, referrerToUse, subFrame.get());
 
+#if ENABLE(CONTENT_EXTENSIONS)
+    if (url.isAboutBlank() && subFrame->page()) {
+        subFrame->page()->protectedUserContentProvider()->userContentExtensionBackend().forEach([&] (const String& identifier, ContentExtensions::ContentExtension& extension) {
```

Comment:
Should use `protectedPage()` here, or pull `subFrame->page()` out into a `RefPtr` before the `if ()` and use it in both places.